stock_picking_report_valued/report/stock_picking_report_valued.xml

```diff
@@ -91,8 +94,7 @@
         inherit_id="stock.stock_report_delivery_has_serial_move_line"
     >
         <xpath expr="//td[@name='move_line_lot_qty_done']" position="after">
-            <t t-if="o.valued and o.sale_id and o.move_line_ids">
-                <t t-if="move_line.picking_id.state != 'done'">
+            <t t-if="move_line.picking_id.state != 'done'">
```

Comment:
This is changing the logic of the report.

Reply:
@pedrobaeza 
Quantity fields' visibility was conditioned by `o.valued`, `o.sale_id` and `o.move_line_ids`, which makes no sense. These fields should always be displayed, not only if the picking fulfills those conditions.

I modified this so that the only columns affected by these conditions are price unit, discount, subtotal, etc.

Reply:
They should be there for a reason. @carlosdauden can you tell us why they are there?

Reply:
The original idea of this module was to allow printing a valued delivery note in the following cases:
- picking not reserved (move_line_ids is empty, get lines from move_lines field product_uom_qty)
- picking reserved, but not done (get quantity from move_line_ids field qty_done or product_uom_qty)
- picking done (get quantity from move_line_ids field product_uom_qty)

If these premises are met, welcome any improvement

Reply:
Then what Carlos describes is complied with these changes, Aritz?

Reply:
True, the module does not work as described at this time in the 15.0 branch.
Since it is modified, would you be so kind as to adjust it to the desired behavior? (or as similar as possible)

Reply:
Hi @carlosdauden  and @pedrobaeza 
As @ao-landoo and @bi-landoo are in holidays this week, I will try to help with this PR.

To simplify the topic, In this PR we will cover only features of outcoming stock pickings.

After deep testing 14 different use cases In last 15.0 OCA version, we only find one small bug.
Bug: "Ordered" label is not printed
Conditions to reproduce: from stock picking in state "Done" and valued "NO", print a delivery slip
![image](https://user-images.githubusercontent.com/13483497/205722835-959b6da2-2c7d-4ebd-ba98-dc577eaed1e3.png)
Today Runboat seems not working so you can test it in our database
https://demo15e.landoo.es/web?db=demo15e_bi_sprv
admin-oca / admin-oca

With PR#244 you can see previous bug is not present and all desired behaviour works as original module, in 3 different states:
1) picking not reserved (move_line_ids is empty, get lines from move_lines field product_uom_qty)
2) picking reserved, but not done (get quantity from move_line_ids field qty_done or product_uom_qty)
3) picking done (get quantity from move_line_ids field product_uom_qty)
You can test in
https://demo15e.landoo.es/web?db=demo15e_bi_sprv_landoo
admin-oca / admin-oca
In this DB you can see module renamed as stock_picking_report_valued_landoo 
![image](https://user-images.githubusercontent.com/13483497/205722782-5662546d-9040-4f55-aebf-3b08e698ed3c.png)

Also we are working to use this module with all features and incoming stock pickings.
Once we test all use cases, we will upload a new PR in next weeks.

I wait for your comments. Many thanks.